.jules/sentinel.md

```
@@ -0,0 +1,4 @@
+## 2025-02-12 - Prevent TOCTOU and DoS in file reading
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)